server/schedule/placement/rule_list.go

```diff
@@ -63,6 +63,27 @@ func (sr *sortedRules) deleteRule(rule *Rule) {
 	}
 }
 
+func checkRulesRaft(rules []*Rule) string {
```

Comment:
Could you add an unit test for this single function?

Reply:
here it is.